object.c

```diff
@@ -909,24 +909,34 @@ rb_class_search_ancestor(VALUE cl, VALUE c)
  *
  * Callback invoked whenever a subclass of the current class is created.
  *
+ * The callback is only called on the immediate superclass of the new
```

Comment:
The word "immediate superclass" is questionable.
When a subclass of `Child` in this example is defined, `Parent.inherited` will be called.
Is `Parent` the "immediate superclass" of that subclass?

Reply:
Correct. This wording is a bit tricky, and I don't like how I phrase it here. At the same time, I'm not sure how to improve it 😅 

My current explanation is focusing on the subclass, and talking about _its_ superclass, but that's not a good fit, because `inherited` is a callback on the superclass (i.e. `self` is the superclass).

I was considering this alternative:

> The system calls this method when a new direct subclass is made.
> If you would like the method to be called for all subclasses, including indirect descendants
> you will need to manually call +super+ in all of the descendants' overrides.

It's written from the perspective of `self` (the superclasses not the subclasses) but it's quite wordy and still not very clear. I'd love any suggestions!

Reply:
Does this look ok now?